tests/test-python-arango-insertion.py

```diff
@@ -0,0 +1,95 @@
+
+import os
+from pathlib import Path
+import argparse
+import cProfile
+import uuid
+from multiprocessing import Queue, Manager
+from urllib.parse import urlparse
+
+from arango import ArangoClient
+
+from wsyntree import log
+from wsyntree.utils import node_as_sexp, strip_url
+from wsyntree.wrap_tree_sitter import TreeSitterAutoBuiltLanguage, TreeSitterCursorIterator
+from wsyntree.tree_models import *
+from wsyntree_collector import arango_collector_worker as wst_arango_worker
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-l", "--language",
+        type=str,
+        help="Language to parse",
+        required=True
```

Comment:
1 Lang at a time?

Reply:
Yes, this file is for evaluating node insert performance so it only accepts / parses one file.